### DISCUSSION TITLE
issue downloading packages

### DISCUSSION THREAD

**Discussion by HananiaW** [giswqs/geog-312#199] [Q&A] [answered]
![Screenshot 2024-12-25 110519](https://github.com/user-attachments/assets/914af6ea-35bb-4b43-a8bc-9d8ceaa9686a)
![Screenshot 2024-12-25 110400](https://github.com/user-attachments/assets/5cbce30b-c78f-4e77-8a87-24de8b420d7d)

When trrying to download the python or mamba packages into the anaconda promt I get these failure messages.
Any idea what I should do?
thanks

**Comment by giswqs** [comment, marked as answer]
Use uv to install packages if mamba doesn't work for you 

Intro to GIS Programming | Week 6: Installing Geospatial Packages Using uv
https://youtu.be/-YnfmHBHkWQ
